### ISSUE TITLE
Dashcam Car Unrecognized - 2018 Grand Cherokee

### ISSUE THREAD

**Issue by atstevenha** [commaai/openpilot#2127]
Dash cam mode car Unrecognized error with LKAS fault after moving. 

<!-

**How to reproduce or log data**

**Expected behavior**

Clear dashcam and LKAS fault: Restart the car to engage. Was able to use open pilot before. It worked without no connection to the OBD 2.
 When would the 2018 Grand Cherokee able to connect to the OBD 2? Thanks in advance 

**Device/Version information (please complete the following information):**
 - Device: [comma two]
 - Dongle ID: 8551ebbdd971b5c8
 
 - Timestamp: When I start the car
 - Version: 0.7.8
 - Car make/model: Jeep Grand Cherokee TrailHawk 2018

**Additional context**

**Comment by atstevenha** [comment]
Please advice. Thank you

**Comment by atstevenha** [comment]
Is something wrong with my post?

**Comment by atstevenha** [comment]
I don't understand what's wrong. I've tried everything and read the entire forum on this issue. This stupid system isn't working since the update.

**Comment by cydia2020** [comment]
is this a duplication of #1996 ?

Did you change branch?

**Comment by atstevenha** [comment]
> is this a duplication of #1996 ?
> 
> Did you change branch?

Thanks for replying.  I haven't changed anything. The cable male is a bit bent but don't think that's it.

**Comment by cydia2020** [comment]
Hi,

In #1996 in the photo you've taken the branch on it clearly showed chffrplus.

https://github.com/commaai/openpilot/issues/1996#issuecomment-670800160

Chffrplus is openpilot's dashcam software and doesn't control the vehicle.

You need to uninstall chffrplus from I think settings - device, someone correct me if I'm wrong here, and during the setup select <b>custom software</b> and put openpilot.comma.ai in the address bar

**Comment by atstevenha** [comment]
> Hi,
> 
> In #1996 in the photo you've taken the branch on it clearly showed chffrplus.
> 
> [#1996 (comment)](https://github.com/commaai/openpilot/issues/1996#issuecomment-670800160)
> 
> Chffrplus is openpilot's dashcam software and doesn't control the vehicle.
> 
> You need to uninstall chffrplus from I think settings - device, someone correct me if I'm wrong here, and during the setup select custom software and put https://openpilot.comma.ai in the address bar
> 
> Edit: the aforementioned steps might not solve your problem, but let's start from here.

That is strange. I'll try it again but I swear I only use https://openpilot.comma.ai/. Thanks!

**Comment by atstevenha** [comment]
I'm starting to think its my cable. Its bent on accident. 
![20200915_105520](https://user-images.githubusercontent.com/69372760/93234512-feeb7480-f741-11ea-82fe-937bb2714432.jpg)

**Comment by cydia2020** [comment]
Yeah looks like it, get a new one

**Comment by adeebshihadeh** [comment]
There was a bug (fixed in a81a18205e85ac576a5cf5e330e3f7b99fcb4152) where community features cars would show the dashcam startup alert if the toggle wasn't enabled. Does your car still fault if you unplug the OBD C cable while leaving the harness intact?

**Comment by atstevenha** [comment]
Thank you everyone for replying back. Chris @ Comma.ai has sent me a new cable, I have tried every cable but it doesn't show the cam so I figured it must be a special cable that isn't just data. Once I receive the cable, Ill update if there are any issues if it works. Chris mention that the 2018 Grand Cherokee can now run the OBD2 without being locked from the security relay. Ill try to have everything plugged in. Do I need to do anything special in the software to get the OBD2 to work? Thanks again, I can't wait to have this working again. =)
